exception-handling/ExceptionHandling/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExceptionHandling;
+
+public class CustomExceptionHandler : IExceptionHandler
+{
+    public async ValueTask<bool> TryHandleAsync(
+        HttpContext httpContext,
+        Exception exception,
+        CancellationToken cancellationToken)
+    {
+        var problemDetails = new ProblemDetails
+        {
+            Status = StatusCodes.Status500InternalServerError,
+            Title = "Custom Exception Handler",
+            Detail = exception.Message
+        };
```

Comment:
problem details looks like something extra. include it in readme.

Reply:
https://learn.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-8.0#problem-details